### PR TITLE
fix: trigger workflow only for first recurring event

### PR DESCRIPTION
**Pull request by CarinaWolli**
## What does this PR do?

This fixes that a workflow with the 'when new event is booked' trigger is only triggered for the first recurring event

### CODE REVIEW

**Review by sean-brydon — CHANGES_REQUESTED** [review]
CleanShot 2024-05-21 at 11 11 23

Hey @CarinaWolli i just tested this and it doesnt seem to work as i am still getting all the emails per scheduled event - am i misunderstanding something

File: packages/features/ee/workflows/lib/reminders/reminderScheduler.ts [hunk 1]
```diff
@@ -141,7 +141,7 @@ export const scheduleWorkflowReminders = async (args: ScheduleWorkflowRemindersA
     calendarEvent: evt,
     isNotConfirmed = false,
     isRescheduleEvent = false,
-    isFirstRecurringEvent = false,
+    isFirstRecurringEvent = true,
```
Comment (CarinaWolli) [inline packages/features/ee/workflows/lib/reminders/reminderScheduler.ts:144]: @Udit-takkar  `isFirstRecurringEvent` should default to `true` if it is normal event and not recurring. Why? The first recurring event is handled the same as a normal event

**Comment by CarinaWolli** [comment]
> CleanShot 2024-05-21 at 11 11 23
> 
> Hey @CarinaWolli i just tested this and it doesnt seem to work as i am still getting all the emails per scheduled event - am i misunderstanding something

@sean-brydon is that a workflow email? looks like the broken integration email

**Comment by sean-brydon** [comment]
> > CleanShot 2024-05-21 at 11 11 23
> > Hey @CarinaWolli i just tested this and it doesnt seem to work as i am still getting all the emails per scheduled event - am i misunderstanding something
> 
> @sean-brydon is that a workflow email? looks like the broken integration email

Sorry for the late reply on this - slipped my inbox. 

I've just tested this again and im seeing the same issue :/ I logged out and send email is still being called N times for the number of recuring events so im not sure its something to do with the email integration 🤔  Will have a dig around

**Comment by CarinaWolli** [comment]
> > > CleanShot 2024-05-21 at 11 11 23
> > > Hey @CarinaWolli i just tested this and it doesnt seem to work as i am still getting all the emails per scheduled event - am i misunderstanding something
> > 
> > 
> > @sean-brydon is that a workflow email? looks like the broken integration email
> 
> Sorry for the late reply on this - slipped my inbox.
> 
> I've just tested this again and im seeing the same issue :/ I logged out and send email is still being called N times for the number of recuring events so im not sure its something to do with the email integration 🤔 Will have a dig around

That's weird. Could you provide the steps how you are testing it? 

That's how I test it:
1. Create a user event type and enable recurring events 
2. Create workflow with 'when new event is booked' trigger and 'email to attendee' action 
3. Set event type active on that workflow 
4. Book the event type with several recurrences
5. I have sendgrid for workflow emails set up, so I check if I received ONE workflow email only

**Comment by sean-brydon** [comment]
> > > > CleanShot 2024-05-21 at 11 11 23
> > > > Hey @CarinaWolli i just tested this and it doesnt seem to work as i am still getting all the emails per scheduled event - am i misunderstanding something
> > > 
> > > 
> > > @sean-brydon is that a workflow email? looks like the broken integration email
> > 
> > 
> > Sorry for the late reply on this - slipped my inbox.
> > I've just tested this again and im seeing the same issue :/ I logged out and send email is still being called N times for the number of recuring events so im not sure its something to do with the email integration 🤔 Will have a dig around
> 
> That's weird. Could you provide the steps how you are testing it?
> 
> That's how I test it:
> 
> 1. Create a user event type and enable recurring events
> 2. Create workflow with 'when new event is booked' trigger and 'email to attendee' action
> 3. Set event type active on that workflow
> 4. Book the event type with several recurrences
> 5. I have sendgrid for workflow emails set up, so I check if I received ONE workflow email only

Well - it turns out there was an issue with my mail setup - had four containers running somehow and they were all receiving emails. 


Works fine now :D Sorry!

**Review by sean-brydon — APPROVED** [review]
LGTM

**Pull request merged**